rmgpy/species.py

```diff
@@ -51,7 +51,9 @@
 
 import rmgpy.quantity as quantity
 from rmgpy.exceptions import SpeciesError, StatmechError
+from rmgpy.molecule.graph import Vertex, Edge, Graph
```

Comment:
Vertex and Edge show up as unused imports for me. If so, can you remove these?

Reply:
Done!